src/kOS/Module/kOSCustomParameters.cs

```diff
@@ -104,6 +104,11 @@ public int InstructionsPerUpdate
                                          "button on it instead of both it and the stock toolbar.")]
         public bool useBlizzyToolbarOnly = false;
 
+        [GameParameters.CustomFloatParameterUI("Default terminal brightness", minValue = 0f, maxValue = 1f, stepCount = 50, displayFormat = "N2",
+                                         toolTip = "Brightness of a kOS terminal when it appears for the first time in a scene.\n" +
+                                         "(You must reload the scene to see the effect of any changes to this slider.)")]
```

Comment:
I chose to do it this way because previously if you chose to adjust the brightness GUI slider on a single terminal window (or use ``set terminal:brightness to something.`` in a script which is the same thing), then closed the terminal window and re-opened it, the setting you chose for that one terminal persisted through the closing and opening of it.  It only reset to the default of 50% when you first instantiate the ``Shared`` object for the CPU, not when you open or close the window again and again.  Thus to preserve that behaviour and not wipe the different individual settings of different terminals whenever you change this ``config`` setting, I kept the fact that it only gets looked at during initializing the ``Shared`` member of the CPU.